github/MainClass.py

```diff
@@ -361,6 +363,22 @@ def get_organizations(self, since=github.GithubObject.NotSet):
             url_parameters,
         )
 
+    def get_enterprise_consumed_licenses(self, login):
```

Comment:
I'd prefer to create an `Enterprise` class that wraps the `/enterprise/{enterprise}` path prefix. That class can provide all endpoints that start with this prefix.

Then, `MainClass` would have only `get_enterprise(enterprise: str) -> Enterprise`.

Returned `Enterprise` instance then has `get_consumed_licenses() -> EnterpriseConsumedLicenses`.

Reply:
Thank you for your quick feedback! Sure, I agree with that, will check and update it.